app.js

```diff
@@ -69,10 +92,17 @@ app.use((err, req, res, next) => {
   console.log(err);
 });
 
+
 const port = process.env.PORT || 3000;
 
+async function startServer() {
+  await require("./db/connect")(process.env.MONGO_URI);
+}
+
```

Comment:
I like the separate function here! We don’t want to initiate a connection twice though, so will want to just call startServer inside the start function